Source/JavaScriptCore/bytecompiler/BytecodeGenerator.h

```diff
@@ -540,6 +540,10 @@ namespace JSC {
 
         RegisterID* emitNode(ExpressionNode* n)
         {
+#if USE(BUN_JSC_ADDITIONS)
+            ExpressionNode* expr = n->isArgumentList() ? static_cast<ArgumentListNode*>(n)->m_expr : nullptr;
+            if (expr && expr->isLinkTimeConstant()) return moveLinkTimeConstant(newTemporary(), static_cast<BytecodeIntrinsicNode*>(expr)->linkTimeConstant());
+#endif
```

Comment:
☝️ With this tweak in `emitNode` we now can pass byteCodeIntrinsics and linkTimeConstants as arguments 😮